cpp/build-support/iwyu/iwyu.sh

```
@@ -42,6 +42,10 @@ affected_files() {
   popd > /dev/null
 }
 
+# Show the IWYU version. Also causes the script to fail if iwyu is not in your
+# PATH
+include-what-you-use --version
```

If IWYU is not in `$PATH`, this prevents a silent failure